src/common/js/timesheet/index.js

```diff
@@ -0,0 +1,244 @@
+          if ($rows.eq(checkRowNo).find('td').eq(COLS.WDAY).text() === '金'){
+            rowspan = checkNextRows + 1;
+          }
+        }
+        // 申請カラム追加
+        var weeklyActionHasColRowspanTag = createWeeklyActionHasColRowspanTag(rowspan,weeklyBuffer);
+        $(this)
+        .addClass('js-rowHasRowspan' + weeklyActionBtnTargetNo)
+        .append(weeklyActionHasColRowspanTag);
+        weeklyActionBtnTargetNo += 1;
+      }
+    }
+
+    // 土日なら、2列追加
+    if( dayOfTheWeek === '土' || dayOfTheWeek === '日'){
+      $(this).append(weeklyActionColTag);
+    }
+
+    if (dayOfTheWeek === '月'){
+
+      // 月曜日の場合は金曜日までを rowspan でグループ化
+      var rowspan = 1;
+      for (var checkNextRows = 1; checkNextRows < 6; checkNextRows++) {
+        var checkRowNo = rowNo + checkNextRows;
+        if (checkRowNo > lastWorkRowIndex){
+          rowspan = checkNextRows;
+          break;
+        }
+        if ($rows.eq(checkRowNo).find('td').eq(COLS.WDAY).text() === '金'){
+          rowspan = checkNextRows + 1;
+          break;
+        }
+      }
+
+      // 申請カラム追加
+      var weeklyActionHasColRowspanTag = createWeeklyActionHasColRowspanTag(rowspan,weeklyBuffer);
+      $(this)
+      .addClass('js-rowHasRowspan' + weeklyActionBtnTargetNo)
+      .append(weeklyActionHasColRowspanTag);
+      weeklyActionBtnTargetNo += 1;
+    }
+
+    // 1行目に「10件毎申請」列を挿入
+    if(rowNo === 1) {
+      var bulkActionHasColRowspanTag = createBulkActionHasColRowspanTag(lastWorkRowIndex,bulkBuffer);
+      $(this).append(bulkActionHasColRowspanTag);
+    }
+
+    // 「週単位申請」ボタンを追加
+    if (0 < weeklyBuffer.length) {
+
+      if (dayOfTheWeek === '金' || isEndOfMonth) {
+
+        $('.js-rowHasRowspan' + (weeklyActionBtnTargetNo - 1))
+          .find('.weekly-action')
+          .append(actionLink(weeklyBuffer, '週単位'));
+
+        weeklyBuffer = [];
+      }
+    }
+
+    // 「10件単位で申請」ボタンを追加
+    if (0 < bulkBuffer.length) {
+      if (bulkBuffer.length === BULKCOUNT || isEndOfMonth) {
+        $('.bulk-action')
+        .append(actionLink(bulkBuffer, '10件毎'));
+        bulkBuffer = [];
+      }
+    }
+
+
+  });
+}
+
+/**
+ * カラムテキスト抽出
+ * @param  {jQueryObject} $cols 対象のtd要素
+ * @param  {Number} i    対象の列番号
+ * @return {String}      td内のテキスト
+ */
+const colText = ($cols, i) => {
+  return $.trim( $cols.eq(i).text() );
+}
+
+/**
+ * 残業時間をクエリ化
+ * TODO: Object.assignにしたい
+ * @param  {Array} param 配列
+ * @return {String}      td内のテキスト
+ */
+const reqParam = (param) => {
+  return $.extend(true, overtimeReportState, {
+    times: JSON.stringify(param),
+    reflect_data_count: param.length - 1
+  });
+}
+
+/**
+ * 残業申請リンク生成
+ * @param  {Array} param リンクに追加するGetパラメータ値
+ * @param  {String} label ボタンのラベル名
+ * @return {String}       生成した挿入するaタグ
+ */
+const actionLink = (param, label) => (
+  $(`<a
+    href="${overtimeReportPath}?${$.param(reqParam(param))}"
+    class="zangyousan__link"
+    target="_blank"
+  >${label}</a>`)
```

Comment:
It seems there is no line break.

![2017-01-28 16 04 55](https://cloud.githubusercontent.com/assets/3190760/22395077/f51e9cca-e575-11e6-91c0-1047496d37cd.png)


Also, with the "zangyousan__link" class,
It seems to be styling with external css,
Is the style stylized when executing bookmarklet?


Please tell me how to check bookmarklets in local environment.


For reference, in the master branch, we can confirm by the following procedure.

1. yarn start
2. bookmarklet / export.min.js and docs / export.min.js are output
3. The path in bookmarklet / export.min.js will be http: //127.0.0.1: 8080 / docs / export.min.js
4. Register bookmarklet / export.min.js in the browser
5. If you do, you can run docs / export.min.js instead of https: //umesan.github.io/overtime-application/export.min.js